docs/user/manual.adoc

```diff
@@ -255,22 +255,9 @@ let g:LanguageClient_serverCommands = {
 ==== YouCompleteMe
 
 1. Install YouCompleteMe by following the instructions
```

Comment:
The enumeration here seems unnecessary now

Reply:
that's correct